plugin/searchindex.vim

```diff
@@ -38,6 +38,8 @@ if !exists('g:searchindex_star_case')
   let g:searchindex_star_case=1
 endif
 
+set shortmess+=s
```

Comment:
Please add a short comment here, it's easier than looking it up in Vim help every time.